app/views/content/ways-to-train.md

```diff
@@ -42,208 +42,151 @@ keywords:
-
-If you do not have a degree, you can do an undergraduate degree which leads to QTS.
-
-Different types of training have different eligibility criteria, [funding options](/funding-your-training) and qualifications. Lots of courses also allow you to study part-time.
-
-You can also [get support to improve your subject knowledge](/improve-your-subject-knowledge) to supplement your training.
-
-Not looking for primary or secondary training? Consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
+You need 'qualified teacher status' (QTS) to work in the majority of schools in England including state maintained primary, secondary and special schools.
 
-$get-a-tta$
+Find out how to get QTS if you:
 
-$pgce$
+* [have a degree](#if-you-have-a-degree)
+* [do not have a degree](#if-you-do-not-have-a-degree)
+* [already have teaching experience](#if-you-already-have-teaching-experience)
 
-PGCE programmes combine academic study with a minimum of 24 weeks spent training in schools.
+You can also find specialised training programmes if you're:
 
-You do not need an academic qualification such as a PGCE to teach in England, but it may be useful if you want to teach in another country.
+* [changing career](#if-youre-changing-career)
+* [have a doctorate](#if-you-have-a-doctorate)
 
-Many PGCE courses give you credits that count towards a master’s degree.
+Not looking to teach in primary or secondary settings? Consider teaching [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
 
-You can also [get funding](/funding-your-training).
+## If you have a degree
 
-#### Postgraduate Diploma in Education
+### Go back to university as a postgraduate student
 
-Some providers offer a Postgraduate Diploma in Education (PGDE) instead of a PGCE. A PGDE sometimes offers more masters credits than a PGCE.
+With university-led training you spend about a third of your time in academic learning and the rest in school placements.
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a PGCE or PGDE <span>course</span></a>
-</p>
+In addition to QTS you'll get a PGCE or PGDE (an academic qualification).
 
-$school-direct-fee-funded$
+Courses typically take one year but you may be able to study part-time.
 
-You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+Fees are usually around £9,250 but you can:
 
-You’ll be based in a school but you will not be employed by them.
+* get [funding that you do not have to pay back](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#bursaries-and-scholarships) for certain subjects
+* apply for a [student loan](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#tuition-fee-and-maintenance-loans)
```

Comment:
These links need to be updated to '/funding-your-training#bursaries-and-scholarships' and '/funding-your-training#tuition-fee-and-maintenance-loans'